data/run.sh

```diff
@@ -58,11 +58,13 @@ if [ ! -d var/db ]; then
   bashio::log "Creating database directory"
   mkdir -p var/db
 fi
-if [ ! -f var/db/db.sqlite3 ]; then
+if [ ! -f var/db/.initialized-v1.0 ]; then
```

Comment:
My only concern with this is that it could cause problems if someone is trying to migrate an existing database into the add-on. But if I ever write the backup, export, import and restore database commands that I've been thinking about for a while, that wouldn't be an issue.